### PR TITLE
ale finished 20

### PR DESCRIPTION
**Pull request by DonStella**

### CODE REVIEW

File: 20-PB-program-multiplechoice/20ale.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        console.log("blue");
+        break;
+    case 3:
+        console.log("green");
+        break;
+    case 4:
+        console.log("yellow");
+        break;
+    default:
+        console.log("try again")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:28]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+console.log("Q2");
+
+let grade = 1; // give in your grade
+
+switch (grade) {
+    case 1:
+        console.log("excellent!")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:37]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    case 2:
+        console.log("great!")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:40]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    case 3:
+        console.log("passed")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:43]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    case 4:
+        console.log("need to study")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:46]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    case 5:
+        console.log("failed")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:49]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    default:
+        console.log("no grade")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:52]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+        break;
+    default:
+        console.log("not a fruit!");
+}
+
+console.log("Q4");
+
+let percentageComplete = null; // type in your percentage
+if (percentageComplete < 30) {
+    console.log("still a long way to go.")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:77]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+} else if (30 >= percentageComplete <= 50) {
+    console.log("slowly getting there")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:79]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+} else if (51 >= percentageComplete <= 81) {
+    console.log("you can do it")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:81]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+} else if (81 >= percentageComplete <= 99) {
+    console.log("you can do it")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:83]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+} else if (percentageComplete = 100) {
+    console.log("Youre there. Well done!")
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:85]: semicolons everywhere :D

File: 20-PB-program-multiplechoice/20ale.js [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+let percentageComplete = null; // type in your percentage
+if (percentageComplete < 30) {
+    console.log("still a long way to go.")
+} else if (30 >= percentageComplete <= 50) {
+    console.log("slowly getting there")
+} else if (51 >= percentageComplete <= 81) {
+    console.log("you can do it")
+} else if (81 >= percentageComplete <= 99) {
+    console.log("you can do it")
+} else if (percentageComplete = 100) {
```
Comment (martl) [inline 20-PB-program-multiplechoice/20ale.js:84]: please use ===

Comment (DonStella) [reply 20-PB-program-multiplechoice/20ale.js]: ok, i added the semicolons and the ===

**Pull request closed without merging**